cpp/include/graph.hpp

```diff
@@ -14,10 +14,39 @@
  * limitations under the License.
  */
 #pragma once
+#include <rmm/device_buffer.hpp>
 
 namespace cugraph {
 namespace experimental {
 
+template <typename T>
+class cugraph_vector {
```

Comment:
I want cugraph_vector to wrap a device_buffer because right now there is a clear way to move the buffer to a column at the python level. I am not sure how an rmm::device_vector can be moved to the column.

And the naming might give the wrong idea but my main aim is to wrap the device_buffer and not provide a full blown vector class and its features.

The wrapper in my opinion should be
- movable
- has the ability to extract a buffer from it
- does not bother the analytic programmer from dealing with rmm device_buffer. Just get pointer and length and be done with it.

Reply:
Shouldn't something like this work?

```
template <typename vertex_t, typename edge_t>
class CSRGraph {
public:
  edge_t* get_offsets() const {
    return reinterpret_cast<edge_t const*>(offsets.data());
  }
private:
  rmm::device_bufffer offsets{};
}
```

Reply:
@seunghwak Yeah. That's what i'll go for in the next push to this PR.

Reply:
I haven't had the time to address this PR but cugraph_vector is to be removed.